src/main/webapp/view/transcript/transcript.js

```diff
@@ -87,6 +91,15 @@ export class TranscriptLineElement extends HTMLElement {
     return hrElement;
   }
 
+  /**
+   * Creates a stylized comment indicator element.
+   */
+  static createCommentIndicator() {
+    const span = document.createElement('span');
```

Comment:
Can you rename this to be more specific?

Reply:
Renamed to be `commentIndicator`, thanks!